Cubical/Foundations/Groupoid.agda

```diff
@@ -0,0 +1,100 @@
+{-
+
+Proof of some groupoid laws for equality
+
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.Groupoid where
+
+open import Cubical.Core.Primitives
+open import Cubical.Core.Prelude
+
+-- shortcut for the composition
+
+_⋆_ : {ℓ : Level} → {A : Set ℓ} → {x y z : A} → (x ≡ y) → (y ≡ z) → (x ≡ z)
+x≡y ⋆ y≡z = compPath x≡y y≡z
+
+infixl 30 _⋆_
+
+-- simultaneaous composition on both sides of a path
```

Comment:
This is ternary composition of the form `p^-1 ; q ; r` right? I found this useful as well when formalizing the Brunerie number: https://github.com/mortberg/cubicaltt/blob/pi4s3_dimclosures/examples/brunerie.ctt#L108

Reply:
We should have a good name for it

Reply:
Yes. Any ideas ?

Reply:
`comp3Path` or `compPath3`? Not the best, but also not the worst...

Alternatively one could have some fancy mixfix name... Like
```_ ^-1 . _ . _```

Where all of the symbols are suitable fancy unicode symbols. I'm not sure how this would interact with the new notation for `compPath` though